### ISSUE TITLE
Page 1 of frontend

### ISSUE THREAD

**Issue by PratikGarai** [micvitc/Interview-Automation-Pipeline#4]
This page will show boxed of all the forms being monitored

**Comment by PratikGarai** [comment]
Okay, so now that the basic structure is ready, can you focus on making the page 1 responsive?

**Comment by Jaisuraj** [comment]
Yeah, responsive is when they fill  the form and after it dynamically gets allotted, it stores the data on the card.

**Comment by PratikGarai** [comment]
Nope, I didn't mean that. What I mean is that if we change the size of the screen, the format shouldn't break.

**Comment by Jaisuraj** [comment]
Okay will look into it
